packages/vite/src/node/build.ts

```diff
@@ -640,3 +644,16 @@ function wrapSsrHook(fn: Function | undefined) {
     return fn.call(this, ...args, true)
   }
 }
+
+function unique(arr: string[]) {
```

Comment:
**thought (non-blocking):** If we don't have something like this already, could try to move it to a `util` file
And then use generic type T

Reply:
I thought about this but I have a slight preference for not implementing something we don't need. But I wouldn't mind going for your suggestion.

Reply:
We already have https://github.com/vitejs/vite/blob/main/packages/vite/src/node/utils.ts

IMO it would be nice here, so in the future, if we need it again, we can directly use it and don't accidentally create duplicate code

Reply:
👍 Done.